estate/models/res_partner.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models,fields
+
+class ResPartner(models.Model):
+    _name = 'res.partner'
+    _inherit = 'res.partner'
+
+    property_ids = fields.One2many("estate.property", "salesman_id", string="Properties", domain=[("state", "in", ["new", "offer_received"])])
```

Comment:
```suggestion
    property_ids = fields.One2many("estate.property", "salesman_id", string="Properties", domain=[("state", "in", ["new", "offer_received"])])
```
can we add multiple conditions in domain, if yes then reply here with updated domain
condition: state should be in 1. new and offer_recieved and 2. bedroom should be more than 3

if domain does not supports multiple domain then how can we handle this?

Reply:
yes we can add multiple conditions to a domain :
property_ids = fields.One2many(
    "estate.property", 
    "salesman_id", 
    string="Properties", 
    domain=[("state", "in", ["new", "offer_received"]), ("bedrooms", ">", 3)]
)
and for more complex situation we can use filtered() incase if domain is not working.